optex/base/optex.lua

```diff
@@ -531,7 +532,11 @@ local function colorize(head, current, current_stroke)
             if current_stroke ~= new and stroke_needed then
                 local stroke_color = token_getmacro("_color-s:"..current)
                 if stroke_color then
-                    newcolor = string_format("%s %s", newcolor or "", stroke_color)
+                    if newcolor then
+                        newcolor = string_format("%s %s", newcolor, stroke_color)
+                    else
+                        newcolor = stroke_color
```

Comment:
Only stroke color will be inserted. Can it happen?

Reply:
Yes, e.g.

```
{\Blue a \vrule}
```
First only non-stroke color is changed, than stroke color is changed.